tests/docker-compose.yml

```diff
@@ -80,8 +85,6 @@ services:
       - server
     ports:
       - 4444
-    extra_hosts:
-      - host.docker.internal:host-gateway
```

Comment:
Missed in https://github.com/johnbillion/plugin-infrastructure/commit/3c1582bc549d8138eeb7195ab9bd01ac1b33d923

Reply:
This is needed for the Chrome container but not the others

Reply:
Reverted.